src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -379,40 +373,47 @@ bool PolicyHandler::ClearUserConsent() {
 }
 
 uint32_t PolicyHandler::GetAppIdForSending() const {
+  LOG4CXX_AUTO_TRACE(logger_);
   POLICY_LIB_CHECK(0);
   const ApplicationSet& accessor =
       application_manager_.applications().GetData();
-  HmiLevelOrderedApplicationList app_list(accessor.begin(), accessor.end());
-
-  LOG4CXX_INFO(logger_, "Apps size: " << app_list.size());
 
-  for (HmiLevelOrderedApplicationList::const_iterator first = app_list.begin();
-       first != app_list.end();
-       ++first) {
-    if ((*first)->IsRegistered()) {
-      const uint32_t app_id = (*first)->app_id();
-      DeviceParams device_params = GetDeviceParams(
-          (*first)->device(),
-          application_manager_.connection_handler().get_session_observer());
+  HMILevelPredicate has_none_level(mobile_api::HMILevel::HMI_NONE);
+  Applications apps_without_none_level;
+  std::copy_if(
+      accessor.begin(),
+      accessor.end(),
+      std::inserter(apps_without_none_level, apps_without_none_level.begin()),
```

Comment:
@dev-gh 
 - `std::back_inserter` returns `std::back_insert_iterator` that uses `Container::push_back()`.
 - `std::inserter` returns `std::insert_iterator` that uses `Container::insert()`
So if we are using `std::vector` - why not to use `std::back_inserter` ?